src/renderers/__tests__/ReactMultiChildText-test.js

```diff
@@ -191,10 +191,20 @@ describe('ReactMultiChildText', () => {
       [true, <div>{1.2}{''}{<div />}{'foo'}</div>, true, 1.2], [<div />, '1.2'],
       ['', 'foo', <div>{true}{<div />}{1.2}{''}</div>, 'foo'], ['', 'foo', <div />, 'foo'],
     ]);
-    expectDev(console.error.calls.count()).toBe(1);
-    expectDev(console.error.calls.argsFor(0)[0]).toContain(
-      'Warning: Each child in an array or iterator should have a unique "key" prop.',
-    );
+    if (ReactDOMFeatureFlags.useFiber) {
+      expectDev(console.error.calls.count()).toBe(2);
```

Comment:
So in this test we call 
```
 ReactDOM.render(<div>{updateWithChildren}</div>, container);
```
 on line 33. With 'fiber' enabled, when 'updateWithChildren' is an array, it's treated as a fragment and the missing keys trigger a warning. This is a separate warning from the one for missing keys in the nested arrays wihin 'updateWithChildren'.

This seems like
A) A corner case
B) Possibly ok?

The only problem I can think of is that it is a change, so folks could have tests which passed before and then fail with Fiber, and that would be confusing.

I'm not sure how to detect the difference and stop the warning if it's a situation like this though.